README.md

```diff
@@ -10,55 +12,43 @@ Migration consists of an analysis part and a code modification part.
 
 Next to the standalone CLI command an add'tl API is provided, which simplifies integration into other tools (e.g. into custom node.js scripts, grunt).
 
-IMNPORTANT: The modified source code needs to be manually reviewed and tested. There is no 100% guarantee that the modified code works as expected.
+**IMPORTANT**: The modified source code needs to be manually reviewed and tested. There is no 100% guarantee that the modified code works as expected.
 
 **If you want to migrate your codebase, please see the [migration guide](./docs/guide/migrationguide.md)**
 
 
 ## Installation
 
-Run the following command to delete the previous installation to avoid conflicts
-```cli
-npm uninstall ui5-migration --global
-```
+### Requirements
+- [Node.js](https://nodejs.org/) (**version 8.5 or higher** ⚠️)
 
 The migration tool is currently available for experimental usage. It can be installed with npm.
 ```cli
-npm install ui5-migration --global
+npm install --global @ui5/migration
 ```
 
 ## Usage
 ### CLI
+
+To verify that the installation worked run:
+```cli
+ui5-migration --help
+```
+
 Execute migration for the current folder:
 ```cli
 ui5-migration migrate
 ```
 
-Currently the CLI is the easiest way to use the ui5-migration tool. Please see [command-line interface](./docs/guide/cli.md) for more details.
-
-### API
-The ui5-migration API might be consumed as in the following:
-
-```js
-var ui5Migration = require('ui5-migration');
-var mySourceCode = "mysourceCodeString";
-var aTasks = undefined;
-var migrationResult = await ui5Migration.migrateString(aTasks, mySourceCode);
-var log = migrationResult.log;
-var myModifiedSourceCode = migrationResult.output;
-```
+Please see [command-line interface](./docs/guide/cli.md) for more details.
 
 
 ### Available migration tasks
 A list of the current migration tasks can be found [here](./docs/guide/tasks.md)
 
-### Available migration output printing options
+### Printing options for migration output
```

Comment:
Just some feedback (no must to change): The word "Printing" seems odd to me here. I'm only thinking of a physical printer when I read this.

Other tools I know of call this "reporting". E.g. [nyc](https://istanbul.js.org/docs/advanced/alternative-reporters/)